config/FileDownlinkCfg.hpp

```diff
@@ -13,11 +13,11 @@ namespace Svc {
     // see if a packet is ready. If it is false, the next packet
     // will be sent as soon as the previous is complete.
     static const bool FILEDOWNLINK_PACKETS_BY_RUN = false;
-    // If this is set to true, if a file is not found the command will fail.
-    // If it is false, the command will still pass.
-    // The false setting is to prevent a sequence from ending with
-    // an error simply because one file name was wrong
-    static const bool FILEDOWNLINK_COMMAND_FAIL_ON_MISSING_FILE = false;
+    // If this is set, errors that would cause FileDownlink to return an error response, such as a
+    // missing file or attempting to send a partial chunk past the end of the file will instead
+    // return success. This is recommended to avoid a non-serious FileDownlink error aborting a
+    // sequence early. These errors will still be logged as events.
+    static const bool FILEDOWNLINK_COMMAND_FAILURES_DISABLED = true;
```

Comment:
Do we really want to default this to true?  It seems that the default should be more restrictive, and open it up for projects to discuss.

Reply:
It previously defaulted to the less restrictive option as well. Since a file downlink error doesn't impact the sate of the rest of the flight software (at least for most use cases!), I don't think most projects would want an error in file downlink to abort a sequence and end an activity prematurely.